CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 6.8-SNAPSHOT
```

Comment:
Given the discussion, this fix should probably be included in a 6.7 patch release.